File: April 19, 2026.py
def get_unique_climbs(steps):
    
    fibonnaci = 0
    n0 = 0
    n1 = 1

    for _ in range(0, steps):
        fibonnaci = n1 + n0

        n0 = n1
        n1 = fibonnaci

    return fibonnaci

File: April 19, 2026.js
function getUniqueClimbs(steps) {

  let fibonacci = 0;
  let n0 = 0;
  let n1 = 1;

  for (let i = 0; i < steps; i++) {

    fibonacci = n1 + n0;
    n0 = n1;
    n1 = fibonacci;
  }

  return fibonacci;
}